### PR TITLE
Update Debugging.md

### PR DESCRIPTION
**Pull request by jpratt2**
## Proposed changes

I would like to add a section about webstorm to the debugging page

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [X] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: website/docs/Debugging.md [hunk 1]
```diff
@@ -109,3 +109,9 @@ Example: [.vscode/launch.json](https://github.com/mgrybyk/webdriverio-devtools/b
 ## Dynamic Repl with Atom
 
 If you are an [Atom](https://atom.io/) hacker you can try [`wdio-repl`](https://github.com/kurtharriger/wdio-repl) by [@kurtharriger](https://github.com/kurtharriger) which is a dynamic repl that allows you to execute single code lines in Atom. Watch [this](https://www.youtube.com/watch?v=kdM05ChhLQE) YouTube video to see a demo.
+
+## Debugging with WebStorm / Intellij
+You can create a node.js debug configuration like this:
+![Screenshot from 2021-05-29 17-33-33](https://user-images.githubusercontent.com/18728354/120088460-81844c00-c0a5-11eb-916b-50f21c8472a8.png)  
+See also https://stackoverflow.com/a/57885521/1691651
```
Comment (christian-bromann) [inline website/docs/Debugging.md:116]: Rather than referring to a SO post, can we bring the information here?

File: website/docs/Debugging.md [hunk 2]
```diff
@@ -109,3 +109,9 @@ Example: [.vscode/launch.json](https://github.com/mgrybyk/webdriverio-devtools/b
 ## Dynamic Repl with Atom
 
 If you are an [Atom](https://atom.io/) hacker you can try [`wdio-repl`](https://github.com/kurtharriger/wdio-repl) by [@kurtharriger](https://github.com/kurtharriger) which is a dynamic repl that allows you to execute single code lines in Atom. Watch [this](https://www.youtube.com/watch?v=kdM05ChhLQE) YouTube video to see a demo.
+
+## Debugging with WebStorm / Intellij
+You can create a node.js debug configuration like this:
+![Screenshot from 2021-05-29 17-33-33](https://user-images.githubusercontent.com/18728354/120088460-81844c00-c0a5-11eb-916b-50f21c8472a8.png)  
+See also https://stackoverflow.com/a/57885521/1691651  
+For more information about how to make a configuration: https://www.youtube.com/watch?v=Qcqnmle6Wu8
```
Comment (christian-bromann) [inline website/docs/Debugging.md:117]: How about:
```suggestion
Watch this [YouTube Video](https://www.youtube.com/watch?v=Qcqnmle6Wu8) for more information about how to make a configuration.
```

**Review by christian-bromann — CHANGES_REQUESTED** [review]

File: website/docs/Debugging.md (hunk 2 above)
Comment (jpratt2) [reply website/docs/Debugging.md:117]: OK

Comment (jpratt2) [reply website/docs/Debugging.md]: Please make the change; I don't know how to accept your suggestion.

**Review by christian-bromann — APPROVED** [review]
👍  Thank you!

**Pull request merged**